base/abstractset.jl

```diff
@@ -215,6 +215,16 @@ false
 ```
 """
 function issubset(l, r)
+
+    rlen = length(r)
+    #This threshold was empirically determined by repeatedly
+    #sampling using these two methods.
+    lenthresh = 70
+
+    if rlen > lenthresh && !isa(r, Set)
```

Comment:
It occurred to me that this should probably use `isa(r, AbstractSet)` (there are other kinds of sets, all of which should probably have efficient `in` methods).

Reply:
Will do. I just learned that `IntSet` is implemented as a bit string.